src/openrct2/Context.cpp

```diff
@@ -717,7 +717,7 @@ namespace OpenRCT2
                 // which the window function doesn't like
                 auto intent = Intent(WC_OBJECT_LOAD_ERROR);
                 intent.putExtra(INTENT_EXTRA_PATH, path);
-                intent.putExtra(INTENT_EXTRA_LIST, const_cast<rct_object_entry*>(e.MissingObjects.data()));
+                intent.putExtra(INTENT_EXTRA_LIST, const_cast<ObjectEntryDescriptor*>(e.MissingObjects.data()));
```

Comment:
I just realized that this is actually a pretty bad idea. Its passing a pointer of a temporary object declared in the catch statement. This is not for this PR but always good re-visiting lines, should raise an issue for this.